day2_homework/anatomical.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Read the lines into an array of floats
+pixel_values = np.loadtxt('anatomical.txt')
+
+# Find the size of a slice over the third dimension
+P = pixel_values.shape[0]/32.0
```

Comment:
Just a reminder that ` something / 32 ` in Python 2 will always give an integer, but will give a floating point value in Python 3.   Doesn't matter in this case, where you know it will be a whole number.